sig-architecture/annual-report-2021.md

```diff
@@ -0,0 +1,102 @@
+
+ - Subproject meetings are smaller, generally in the half dozen range.
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?**
+
+ - We have a rotating schedule of subproject updates in the bi-weekly community meeting. Our OWNERS files are up-to-date as well.
+
+**How does the group get updates, reports, or feedback from working groups? Are there any springing up or being retired? Are OWNERS files up to date in these areas?**
+
+ - We have not been getting updates from the working groups on any regular cadence.
+ - We lost track of the WGs we sponsored. We will make it a priority to get updates in the next upcoming meetings.
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+ - [June, 2020](https://youtu.be/ObqQxRRl9RQ?t=2277) at Kubernetes Community Meeting. [Slides](https://docs.google.com/presentation/d/1NytMrpVYKzFo7rLcEEHnFl8zOx05fnjs3xBSZXVE0nI/edit?usp=sharing).
+ - [December, 2020](https://youtu.be/rnNqcUeCD8E) at KubeCon NA, published on YouTube. We do not have anything lined up for Kubecon EU. We will prioritize giving our next update in Kubecon NA.
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+- All SIG chairs are active and attend most bi-weekly full SIG meetings. We do not have separate tech leads.
+
+- For the conformance subproject, all owners are active in PR review and meeting attendance.
+
+- For the production readiness subproject, all owners are active in both KEP reviews and meetings.
+
+- For the enhancements subproject, all owners are all active.
+
+- For the API review subproject. There is no meetings as such. The subproject coordinates via a project board.
+
+- For the code organization subproject, @dims and @liggitt run regular meetings and review PRs and issues, pulling in other owners as needed.
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+- Mailing list members, though based on bounces during calendar updates, there are quite a few out-of-date memberships.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+- PRR assigns specific approvers to each KEP that is targeting a release. Therefore, we have a clear picture of how much each
+  approver must do each cycle. In 1.21, each approver reviewed 22 KEPs. for the 1.22 cycle, we have a new approver (@ehashman) to
+  help reduce the load on each approver. Go Elana!
+- Code Organization is coming along. Though we haven't updated dep-approvers in a while. Currently the root OWNERS are a bottleneck as their time is limited for dependency updates. We have a new approval plugin that wiiiii
```

Comment:
wiiii what

Reply:
was supposed to be "will help" :)

Reply:
cc @nikhita 
as an FYI

Reply:
* https://github.com/kubernetes/test-infra/issues/7690 would allow routing dep approval to a distinct group
* started dep-reviewers alias in https://github.com/kubernetes/kubernetes/pull/101670, can use to get additional help on reviews

Reply:
> kubernetes/test-infra#7690 would allow routing dep approval to a distinct group

@ykakarap is going to demo this in tomorrow's SIG Testing meeting :smile: so this should hopefully be resolved soon!